locales/en/create-kafka-instance.json

```
@@ -7,8 +7,10 @@
   "client_connections_value": "up to {{value}}",
   "cloud_provider": "Cloud provider",
   "cloud_region": "Cloud region",
+  "cloud_region_description": "More cloud regions are available with a subscription.",
   "connection_rate_value": "up to {{value}} connections/second",
   "create_instance": "Create instance",
+  "create_instance_title": "Create a Kafka instance",
```

```suggestion
  "create_instance_title": "Create a $t(common:kafka) instance",
```

This is to avoid hardcoding Kafka everywhere and use a glossary of common terms instead.